client/src/components/linked-story/linked-story-connector.js

```diff
@@ -3,6 +3,22 @@ import PropTypes from 'prop-types';
 import * as actions from '../../redux/actions';
 import LinkedStoryDragWrapper from './linkedstory-drag-wrapper';
 
+const mapStateToProps = (state, props) => {
+  const { members } = state;
+  const {
+    story: { owner_ids = [] },
+  } = props;
+
+  const ownerNames = owner_ids.map(id => {
+    const member = members.byId[id];
+    if (!member) return null;
```

Comment:
This feels dangerous to have an array that could contain `null`. I think you should either filter them out with a following `.filter(x => x !== null)` call or turn this `.map()` into a `.reduce()` and save on the extra iteration.

Reply:
Oh yeah, I think when I wrote this I wasn't planning on keeping null items in the array, using `reduce` is a good idea.